AdyenComponents/Apple Pay/ApplePayComponentExtensions.swift

```diff
@@ -35,6 +45,7 @@ extension ApplePayComponent: PKPaymentAuthorizationViewControllerDelegate {
                                       shippingContact: shippingContact)
         
         submit(data: PaymentComponentData(paymentMethodDetails: details, amount: self.amountToPay, order: order))
+        completion(.success)
     }
 }
 
```

Comment:
I think these change breaks ApplePayComponent flow when used as an individual component, and tbh I am not even sure its fully working if used in DropIn